ml-agents/mlagents/utils/load_from_hf.py

```diff
@@ -0,0 +1,36 @@
+import os
+import logging
+import argparse
+from huggingface_hub import HfApi, snapshot_download
+
+def load_from_hf(repo_id: str, 
+                 local_dir: str):
+    """
+    Download a model from Hugging Face Hub.
+    :param repo_id: id of the model repository from the Hugging Face Hub
+    :param local_dir: local destination of the repository
+    """
+    _, repo_name = repo_id.split('/')
```

Comment:
```suggestion
    repo_name = repo_id.split('/')[-1]
```

(nit) just in case it's a canonical repo like "gpt2" (I'm thinking of integrating your code to the huggingface-cli so let's handle it :D)

Reply:
We only have new ML Agents, so having old canonical is not an option